core/cla_types.go

```diff
@@ -0,0 +1,18 @@
+package core
+
+// CLAType is one of the supported Convergence Layer Adaptors
+type CLAType uint
```

Comment:
This type does not really fit into the `core` package. If you want to pull it up, maybe the `cla` package is a better match.

Reply:
Followup, why did you even remove it from the `discovery` package?

Reply:
Because, if it's in the discovery package, you get a cyclic import from `core` importing `discovery` and `discovery` importing `core`